provider/mock/mock.go

```diff
@@ -354,8 +354,8 @@ func (p *MockV0Provider) GetPods(ctx context.Context) ([]*v1.Pod, error) {
 	return pods, nil
 }
 
-func (p *MockV0Provider) ConfigureNode(ctx context.Context, n *v1.Node) {
-	ctx, span := trace.StartSpan(ctx, "mock.ConfigureNode")
+func (p *ProviderV0) ConfigureNode(ctx context.Context, n *v1.Node) {
+	_, span := trace.StartSpan(ctx, "mock.ConfigureNode")
```

Comment:
This is somewhat of an aside, but why do all of the other spans just have the function name, whereas this is mock.ConfigureNode?

Reply:
🤷‍♂

Reply:
Updated to make this like everything else.